package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "next",
-  "version": "6.0.3-canary.1",
+  "version": "6.0.3",
```

Comment:
🤔 Sounds like you created from the wrong branch.

Reply:
@timneutkens Hmmm I forked the repo, cloned that to my local machine...did a get fetch upstream to make sure my canary was the latest, and then branched off that. Did I miss a step somewhere?

Reply:
@TheRobBrennan try doing `git pull origin canary` where origin is zeit/next.js. Besides this I want to merge the PR 😄